addons/hr_recruitment/views/hr_recruitment_views.xml

```diff
@@ -386,7 +411,7 @@
         <field name="res_model">hr.recruitment.source</field>
         <field name="view_mode">tree</field>
         <field name="domain">[('job_id', '=', active_id)]</field>
-        <field name="context">{'default_job_id': active_id}</field>
+        <field name="context">{'default_job_id': active_id,'default_source_id':3}</field>
```

Comment:
**Never hardcode IDs**.
IDs are auto-incremented and are never guaranteed to be given to a specific record.
It means if we had another record before the one you want in the demo data, it will take the "3" ID and your development will be broken.

You can use an "eval" and "ref" to find the matching default source you want and achieve this.
I let you search a bit on that, there are examples in the code base.